### DISCUSSION TITLE
Microsoft authenticate failed: Invalid credentials

### DISCUSSION THREAD

**Discussion by Lotsofcake** [ORelio/Minecraft-Console-Client#2363] [General]
I have tried to login to multiple accounts multiple times using the correct password and every time I have gotten this same result. I also turned on debugmessages and I guess i can send what that printed. 

Also, this is unrelated, but once i log in one account how do i log in multiple, will I be prompted to login again, or? 

 action: send /hello
[MCC] [ScriptScheduler] Loaded task:
triggeronfirstlogin: False
 triggeronlogin: True
 triggerontime: False
 triggeroninterval: False
 timevalue:
 timeinterval: 56
 action: send /login pass
Color test: Your terminal should display [0123456789ABCDEF]: (4bit)[0123456789ABCDEF]
Color test: Your terminal should display [0123456789]: (8bit)[0123456789]
Color test: Your terminal should display [0123456789]: (24bit)[0123456789]
[SessionCache] Initializing FileSystemWatcher for file: C:\Users\attic\Downloads\MinecraftClient-windows-x64\SessionCache.ini
Loading Minecraft profiles: launcher_profiles.json
No sessions could be loaded from disk
Login :
Password(invisible): {0}
Connecting to Microsoft...
< GET /oauth20_authorize.srf?client_id=000000004C12AE6F&redirect_uri=https://login.live.com/oauth20_desktop.srf&scope=service::user.auth.xboxlive.com::MBI_SSL&display=touch&response_type=token&locale=en HTTP/1.1
< Host: login.live.com
< User-Agent: Mozilla/5.0 (XboxReplay; XboxLiveAuth/3.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36
< Accept: */*
< Connection: close
<
<
< POST /ppsecure/post.srf?client_id=000000004C12AE6F&contextid=93A7BFDDB251EF4E&opid=3D0F4F1AF7094A26&bk=1670517374&uaid=87afba821bd34fb784910bf3c4868453&pid=15216 HTTP/1.1
< REDACTED
< User-Agent: Mozilla/5.0 (XboxReplay; XboxLiveAuth/3.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36
< Accept: */*
< Connection: close
< Content-Type: application/x-www-form-urlencoded
< Content-Length: 295
<
< REDACTED
Status code: 200
Headers:
  cache-control: no-store, no-cache
  pragma: no-cache
  content-type: text/html; charset=utf-8
  expires: Thu, 08 Dec 2022 16:35:14 GMT
  p3p: CP="DSP CUR OTPi IND OTRi ONL FIN"
  x-frame-options: DENY
  x-dns-prefetch-control: on
  link: <https://logincdn.msftauth.net>; rel=preconnect; crossorigin,<https://acctcdn.msauth.net>; rel=preconnect; crossorigin,<https://acctcdn.msftauth.net>; rel=preconnect; crossorigin,<https://acctcdn.msauth.net/>; rel=dns-prefetch,<https://acctcdn.msftauth.net/>; rel=dns-prefetch,<https://acctcdnmsftuswe2.azureedge.net/>; rel=dns-prefetch,<https://acctcdnvzeuno.azureedge.net/>; rel=dns-prefetch,<https://logincdn.msauth.net/>; rel=dns-prefetch,<https://logincdn.msftauth.net/>; rel=dns-prefetch,<https://lgincdnvzeuno.azureedge.net/>; rel=dns-prefetch,<https://lgincdnmsftuswe2.azureedge.net/>; rel=dns-prefetch
  referrer-policy: strict-origin-when-cross-origin
  x-ms-route-info: R3_BAY
  x-ms-request-id: bd02bcff-acfa-4f8d-911a-8787529eba48
  ppserver: PPV: 30 H: BY1PPF21250BC56 V: 0
  x-content-type-options: nosniff
  strict-transport-security: max-age=31536000
  x-xss-protection: 1; mode=block
  set-cookie: MSPOK= ; expires=Thu, 30-Oct-1980 16:00:00 GMT; domain=login.live.com; Secure; path=/; REDACTED
  date: Thu, 08 Dec 2022 16:36:14 GMT
  connection: close
  content-length: 23487

Cookies:
  REDACTED
Body: (Truncated to 200 characters)
<!DOCTYPE html><!-- PreprocessInfo: CBA-1108_004314:SA2PNPF0000222D, ...

Microsoft authenticate failed: Invalid credentials. Check your credentials
    at MinecraftClient.Protocol.ProtocolHandler.MicrosoftMCCLogin(String email, String password, SessionToken& session)

Login failed : Incorrect username/password, blacklisted IP or too many logins.

Not connected to any server. Use '/help' for help.
Or press Enter to exit Minecraft Console Client.`

**Comment by BruceChenQAQ** [comment]
Try to toggle `Method = "browser"` in the settings and login once.
